client/shared/src/main/scala/org/http4s/client/DefaultClient.scala

```diff
@@ -26,6 +26,7 @@ import fs2.Stream
 import org.http4s.Status.Successful
 import org.http4s.headers.Accept
 import org.http4s.headers.MediaRangeAndQValue
+import cats.data.NonEmptyList
```

Comment:
organize imports is about to bite your head 😛

Reply:
Aww shucks, I ran `prePR`

Reply:
Hmm yeah, I had to run `scalafixAll` manually.